.github/workflows/ci.yml

```
@@ -66,7 +66,15 @@ jobs:
             ${{ runner.OS }}-node-
             ${{ runner.OS }}-
       - run: npm ci --legacy-peer-deps
+      - name: Configure Datadog Test Visibility
+        uses: datadog/test-visibility-github-action@v1
+        with:
+          languages: js
+          service: gogovsg
+          api_key: ${{ secrets.DD_API_KEY }}
       - run: npm run test
+        env:
+          NODE_OPTIONS: -r ${{ env.DD_TRACE_PACKAGE }} --import ${{ env.DD_TRACE_ESM_IMPORT }}
```

y'all using jest, so don't need the `--import`; might break something

```suggestion
          NODE_OPTIONS: -r ${{ env.DD_TRACE_PACKAGE }}
```